config/testV5/config.js

```diff
@@ -167,4 +167,11 @@ module.exports = {
 		},
 		sender: '"3D Repo" <dummyEmail@3drepo.org>'
 	},
+
+	sso: {
+		aad: {
+			clientId: "4caba479-bfa6-46bc-9214-a0dc1dcbf8c7",
```

Comment:
unless these are keys that AAD let everyone use, i don't think we should be putting genuine ids here (even if it's just a sandbox application.